src/config/urls.py

```diff
@@ -20,7 +20,8 @@
     path("games/", include("chigame.games.urls")),
     # Django-machina forum
     # https://django-machina.readthedocs.io/en/latest/getting_started.html#urls-configuration
-    path("forum/", include(machina_urls)),
+    path("forums/", include(machina_urls)),
```

Comment:
Standardizing this is also good!

Reply:
Be aware that making this change will require updating any existing references to the original URL scheme when  this is merged. As far as I can tell that is currently just [one line](https://github.com/uchicago-cs/chigame/blob/721ef602d22a7058f7f845665b2bb5759fd25307/src/templates/base.html#L72) though. 

Additionally, while it shouldn't be an issue, it's also worth keeping in mind that this scheme would no longer match the configuration given in the django-machina [installation instructions](https://django-machina.readthedocs.io/en/latest/getting_started.html#urls-configuration).

Reply:
navbar. the navbar will no longer work if you change this URL since root forums URL in the `config/urls.py` is not namespaced:

```py
    # Django-machina forum
    # https://django-machina.readthedocs.io/en/latest/getting_started.html#urls-configuration
    path("forums/", include(machina_urls)),
    path("forums/", include("chigame.forums.urls")),
```

compared to: 
```py
    path("", TemplateView.as_view(template_name="pages/home.html"), name="home"),
    path("about/", TemplateView.as_view(template_name="pages/about.html"), name="about"),
```